rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc

```diff
@@ -0,0 +1,68 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
+
+Use the Activity Library Explorer to edit, copy, or delete activities and search for activities in your organization's library based on Activity Tags, which you can create and assign to the activities to further categorization and filtering capabilities. 
+
+== Before you Begin 
+
+//Update this when IAM is implemented.
+* To open and to use the Activity Library, you need the *Activity Library Open* and *Activity Library Administration* privileges. 
++
+For more information about assigning user privileges, see xref:rpa-manager::usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
+
+== Upload an Activity to the Activity Library
+
+To upload an activity to the Activity Library:
+
+. In RPA Builder, open a process for editing. 
+. Click an activity to select it and open the context pad. 
+. In the context pad, click *Add to Activity Library* (image:add-to-activity-library-icon.png["The Add to Activity Library Icon.", 2%, 2%])
++
+image:context-pad.png["The context pad of an activity.", 50%, 50%]
+. In the *Upload Workflow to Activity Library* window that opens, provide a name and a description for the activity. 
++
+image:upload-workflow-to-activity-library.png["The Upload Workflow to Activity Library window.", 50%, 50%]
++
+[IMPORTANT]
+Each activity must have a unique name, which cannot be longer than 60 characters. Otherwise, RPA Builder cancels the upload and shows an error message. 
++
+. Click *Upload*. 
+
+== Import an Activity from the Activity Library 
+
+To import an activity from the Activity Library to your project: 
+
+. Select an activity in your project. 
+. In the context pad, click *Import Workflow from Activity Library* (image:import-workflow-from-activity-library-icon.png["The Import Workflow from Activity Library Icon.", 2%, 2%])
+. In the window that opens, select the activity to import and click either *Add as a library reference* or *Add as a local copy*, depending on which method you want to use. 
+
+** *Add as a library reference*
++
+This method creates in each process a link to an activity from the Activity Library. Because the activity is already saved on the library, it is not uploaded again. The activity automatically receives the name of the reference, and you cannot change the name in the BPMN editor. An activity with a referenced workflow shows a gray background in the task element. 
```

Comment:
```suggestion
This method creates a link in each process to an activity from the Activity Library. Because the activity is already saved in the library, it is not uploaded again. The activity automatically receives the name of the reference, and you cannot change the name in the BPMN editor. An activity with a referenced workflow shows a gray background in the task element. 
```